tests/regression_tests/test_routes_dijkstra.py

```diff
@@ -142,13 +141,14 @@ def calculate_dijkstra_route(config, mesh):
 
     # Initial set up
     waypoints   = extract_waypoints(mesh)
-    
+
     # Calculate dijkstra route
-    rp = RoutePlanner(mesh, config, waypoints)
-    rp.compute_routes()
+    rp = RoutePlanner(mesh, config)
+    routes = rp.compute_routes(waypoints)
     
     # Generate json to compare to old output
-    new_route = rp.to_json()
+    new_route = mesh
+    new_route['paths'] = routes[0].to_json()
```

Comment:
This will only hold for a `test_routes_dijkstra.py` that returns a single path (`routes[0]`) is this fine ?

Reply:
I'll double check this

Reply:
I've changed this to work with multiple paths, if necessary